packages/components/src/components/primitives/Snackbar/DefaultSnackbar/DefaultSnackbar.styles.ts

```diff
@@ -0,0 +1,28 @@
+import { SnackbarType } from './DefaultSnackbar';
+
+export const getStyle = ({
+  isTextRightPosition = false,
+  type = SnackbarType.Default,
+}) => ({
```

Comment:
these parameters will not be omitted, we should not use an object in this case

Reply:
Removed object